ui/src/app/pages/access/maybe-module.tsx

```diff
@@ -0,0 +1,80 @@
+// Renders a module when it's enabled via feature flags.  Returns null if not.
```

Comment:
I suspect that the MaybeModule may be able to abstracted away, but I did not pursue that for this PR.